schemas/webledger-validator.js

```diff
@@ -34,7 +34,9 @@ const createOperation = {
       enum: ['CreateWebLedgerRecord'],
     },
     record: {
-      required: ['@context', 'id'],
+      // FIXME: IS THIS CORRECT?
+      // required: ['@context', 'id'],
+      required: ['id'],
```

Comment:
Yes, I think this is correct. It no longer has to be JSON-LD. Of course, specific ledgers may have more stringent requirements.